javascript/src/test/pages/AnalyzeMessageData/CountMessagesByUser.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import CountMessagesByUser from 'main/pages/AnalyzeMessageData/CountMessagesByUser';
+import aggregateUserMessageCount from 'main/utils/aggregateUserMessageCount';
+describe('Count Messages By User tests', () => {
+
+  test('renders without errors', () => {
+    render(<CountMessagesByUser />);
+  });
+
+  test('array test', () =>{
+    var message = {user_profile: {real_name: 'Jane Doe'}}; 
+    var message2 = {user_profile: {real_name: 'John Doe'}}; 
```

Comment:
Same as an above comment, we should try to avoid using `var` in JavaScript and instead use `let` and `const` where appropriate. Also there's a random character at the end of the line, ` ` that I don't recognize, could we look into getting this removed?